dev_guide/application_lifecycle.adoc

```diff
@@ -0,0 +1,88 @@
+= Application Life Cycle Examples
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+As a PaaS, OpenShift is designed for building and deploying applications.
+Depending on how much you want to involve OpenShift in the development process,
+you can choose to develop on OpenShift and use it to continuously develop an
+application, or you can deploy a fully developed application onto a OpenShift
+instance.
+
+== Developing on OpenShift
+
+You can develop your application on OpenShift directly. Use the following if you
+plan to use OpenShift as a method to build and deploy your application:
+
+Initial Planning::
+* What does your application do?
+* What programming language will it be developed in?
+ 
+Access to OpenShift::
+* OpenShift should be installed by this point, either by yourself or an
+administrator within your organization.
+
+Develop::
+* Using your editor/IDE of choice, create a basic skeleton of an application. It
+should be developed enough to tell OpenShift what kind of application it is.
+* Push the code to your git repository.
+
+Generate::
+* link:../dev_guide/new_app.html[Create a basic application]. Here, OpenShift
+generates build and deployment configurations.
```

Comment:
"Here, " is somewhat unsmooth unless i put on my hacker hat.

Suggest: "Create a basic app." (no link) followed by "The new-app command generates..." w/ "new-app" being linked.